mppm_versioned_docs/version-1.0.0/about.md

```diff
@@ -7,10 +7,7 @@ description: Overview of Multiplayer Play Mode
 Use Multiplayer Play Mode to test multiplayer functionality within the Unity Editor. You can simulate up to four Players (the main Editor Player and three Virtual Players) simultaneously on the same development device while using the same source assets on disk. Multiplayer Play Mode can help you create multiplayer development workflows that reduce project build times, run it locally, and test the server-client relationship.
 
 ## Compatibility
-Multiplayer Play Mode version 1.0.0 is compatible with Unity Editor versions 2023.1 or later.
-
-## Compatibility
-Multiplayer Play Mode version 1.0.0 is compatible with Unity Editor versions 2023.1 or later.
+Multiplayer Play Mode version 1.0.0 is compatible with Unity Editor versions 6000.0.3f1 or later.
```

Comment:
Not sure why this carried forward...

Reply:
You tend to pick up random/other commits if you're creating your feature branch from `main` rather than `develop`.